 <launch>
<!--parent-->
<arg name="robot_base_frame_name"          default="base_footprint"/>
<!--child-->
<arg name="HD_frame_name"          default="omni_rotation"/>

<!--Rotation matrix input from robot_base_frame_name(Parent) to HD_frame_name(child)-->
<!--        [11,12,13]                                          -->
<!-- R ===  |21,22,23|  ====    "11 12 13 21 22 23 31 32 33"    -->
<!--        [31,32,33]                                          -->
<arg name="rot_matrix_array" default= "0 0 -1 -1 0 0 0 1 0"/> <!--MUST BE SPACE SEPERATED!!-->
<!--<arg name="rot_matrix_array" default= "0 -1 0 0 0 1 -1 0 0"/>-->

    <node pkg="haptic_device_rotation" name="haptic_device_rotation" 
    type="haptic_device_rotation.py" output="screen">

    <param name="robot_base_frame_name"     value="$(arg robot_base_frame_name)"/>
    <param name="HD_frame_name"     value="$(arg HD_frame_name)"/>

    <param name="rot_matrix_array" value="$(arg rot_matrix_array)"/>

    </node>
 </launch>